### DISCUSSION TITLE
Welcome to PowerMenu Discussions!

### DISCUSSION THREAD

**Discussion by skydoves** [skydoves/PowerMenu#67] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zahidcataltas** [comment]
Very successful work, congratulations :) Is it possible for the menu expansion direction to be bottom-up?
